dashboard/app/models/unit.rb

```diff
@@ -2129,7 +2129,8 @@ def ai_assessment_enabled?
   end
 
   def show_ai_assessments_announcement?(user)
-    user&.teacher? && ai_assessment_enabled? && !user.has_seen_ai_assessments_announcement?
+    # limit to CSD to avoid showing on allthethings
+    user&.teacher? && in_initiative?('CSD') && ai_assessment_enabled? && !user.has_seen_ai_assessments_announcement?
```

Comment:
this is a shortcut to avoid having to update many UI tests which use /s/allthethings.